ballerina/document-types.bal

```diff
@@ -84,3 +84,12 @@ public type ImageDocument record {|
     # Image content, either a URL or binary data
     Url|byte[] content;
 |};
+
+# Represents a chunk of image within a image document.
+public type ImageChunk record {|
```

Comment:
We don't have to introduce image chunk for now.

Reply:
+1